BusinessLogic/Transactions/TransactionAgent/TransactionAgent.cs

```diff
@@ -1,7 +1,11 @@
+using Newtonsoft.Json;
+
 namespace baseledger_replicator.BusinessLogic.Transactions.TransactionAgent;
 
 public class TransactionAgent : ITransactionAgent
 {
+    // TODO: Should we be thinking about extracting urls and accessing them via options or something like that?
+    // Should we be introducing replicatorService for direct communication with replicator node or it's fine to invoke from the agent?
     private readonly string url = "http://0.0.0.0:1317/cosmos/tx/v1beta1/txs/";
```

Comment:
split url into "http://0.0.0.0:1317" and then add the rest of the url in respective agent methods

Reply:
what is the full url for creating transaction?